offline_cam/data_loading.hpp

```diff
@@ -0,0 +1,88 @@
+#include "csv_iterator.hpp"
```

Comment:
same as csv_iterator.hpp

Reply:
The data_loading file is different for imu and cam, so it's better that we keep one for each folder.